plugins/slack_app/app/models/slack_identifier.rb

```diff
@@ -0,0 +1,8 @@
+class SlackIdentifier < ActiveRecord::Base
+  belongs_to :user
+
+  def self.app_token
+    record = SlackIdentifier.find_by_user_id nil
```

Comment:
sounds ok ... maybe document this usecase/flow a bit

On Sat, Jul 9, 2016 at 5:47 AM, Ben Straub <notifications@github.com> wrote:

> In plugins/slack_app/app/models/slack_identifier.rb
> <https://github.com/zendesk/samson/pull/1130#discussion_r70166707>:
>
> > @@ -0,0 +1,8 @@
> > +class SlackIdentifier < ActiveRecord::Base
> > +  belongs_to :user
> > +
> > +  def self.app_token
> > +    record = SlackIdentifier.find_by_user_id nil
>
> Yeah, this is a bit of a hack. We need a Slack API token in order to post
> chat messages, and I'm storing that in this table with user_id = nil. I'd
> like this to be smooth when the Slack admin authorizes the app to the team,
> and not stop the whole process until she can get an ops person to change
> the server configuration to store this value. Is there a better way?
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1130/files/2240ef77f3386f8e2f818a32d96356c4a741e711#r70166707>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ9ZaX1SPXPC1Joa_Ybo6MTZecLfmks5qT5hvgaJpZM4JIRHk>
> .
>